guides/hidden-pages.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to add hidden pages"
+description: "Exclude pages from your docs.json"
+---
+
+Sometimes you may want to write a page and not publish it immediately.
+
+For content that is not strictly confidential, but isn't ready for prime time, you can add pages to your docs without referencing them in your `docs.json`. This is what is considered a "hidden page".
+
+As long as the page is not described under any "group", "tab" or other element in the navigation section of your `docs.json`, the only way a user will be able to navigate to the page is if they know or guess the page URL.
+
+<Note>
+Some navigation elements like sidebars, dropdowns, and tabs may shift or appear empty on hidden pages.
+</Note>
+
+If you want hidden pages to be indexed for SEO purposes and don't mind if their content appears in search results, you can add the following setting to your `docs.json`:
+
+```
+"seo" {
+    "indexing": all
+}
+```
+
+The settings for SEO indexing also influence whether the contents of hidden pages can appear when your docs are searched internally with the search bar. If you want to make sure hidden pages' content does not appear in SEO or internal search results, make sure to add `noindex: true` to the frontmatter of your page.
+
+Learn more about SEO and Sitemaps configuration in the [SEO](https://mintlify.com/docs/settings/seo#sitemaps) docs.
+
+
+If you have certain pages that you only want exposed to specific groups of your own users, you may want to use the selective groups access available through [Personalization](https://mintlify.com/docs/settings/authentication-personalization/personalization#showing-hiding-pages).
+
+If you have docs that you want under no circumstances exposed to the wider web, you may want to use [Authentication](https://mintlify.com/docs/settings/authentication-personalization/authentication).
```

Comment:
```suggestion
If you have pages that you want only specific groups of your own users to have access to, you can use the selective groups access available through [Personalization](/settings/authentication-personalization/personalization#showing-hiding-pages).

If you have pages that you want to maintain strict access control to, you can use [Authentication](/settings/authentication-personalization/authentication).
```

Let's cut the first link since if people are interested, they should be able to find it and the second two links are probably more likely to be related to tasks a user is thinking of while reading up on hidden pages. Just a few wording edits and changed the links to relative links.